`npm install -g gh-pages` failed in `cimg/node:current` image

We are using `cimg/node:current` as base image for one of the CI pipeline. Today it failed without any incoming code changes on our end. The specific failed command is `npm install -g gh-pages`. The error traceback is:

```
npm ERR! code EACCES
npm ERR! syscall mkdir
npm ERR! path /usr/local/lib/node_modules/gh-pages
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/gh-pages'
npm ERR!  [Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/gh-pages'] {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'mkdir',
npm ERR!   path: '/usr/local/lib/node_modules/gh-pages'
npm ERR! }
npm ERR! 
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR! 
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/circleci/.npm/_logs/2022-07-29T00_54_08_567Z-debug-0.log


Exited with code exit status 243
```

I tried to ssh into CI container and run this command with admin permission, i.e. `npm install -g gh-pages` and it worked. However, in your doc it's not encouraged to do so. I'm hoping you could provide some insights here. Thanks.


Attached below is all the steps happen before the failed command:

``` 
update-docs:
    docker:
      - image: cimg/node:current
    steps:
      - attach_workspace:
          at: /tmp/workspace/
      - checkout
      - run:
          name: Copy all docs into repo
          command: |
            cp -r /tmp/workspace/docs/* docs/
      - run:
          name: Install gh-pages utility
          command: |
            npm install -g gh-pages
```

--- a/README.md
+++ b/README.md
@@ -43,8 +43,6 @@ In the above example, the CircleCI Node.js Docker image is used for the primary 
 More specifically, the tag `15.0.1` is used meaning the version of Node.js will be Node.js v15.0.1.
 You can now use Node.js within the steps for this job.
 
-We recommend not using `sudo` with npm as it is not best practice, not needed with this image, and would not work. e.g `npm install -g <packagename>` will yield the same results as running with sudo. Additionally, nvm works using shell and user modification for the default `circleci` user, however, this is not carried over to sudo or root
-
 ## How This Image Works
 
 This image contains the Node.js programming language and its package managers.

--- a/README.md
+++ b/README.md
@@ -48,6 +48,8 @@ You can now use Node.js within the steps for this job.
 This image contains the Node.js programming language and its package managers.
 This includes `npm` and `yarn`.
 
+There was a recent change reimplmenting vanilla node after briefly using nvm. While it is implied and recommended to not use `sudo` when possible, you will now need it when installing packages globally.
+
 ### Variants
 
 Variant images typically contain the same base software, but with a few additional modifications.

